### PR TITLE
Update interface

### PR DESCRIPTION
**Pull request by NekoSelena**
- [x] Add filter buttons
- [x] Replace label with TextView ( fix issue #68 )
- [x] Make the filters work

### CODE REVIEW

**Review by huntears — CHANGES_REQUESTED** [review]
The log view scrolls down to the bottom all the time, it is impossible to check older logs than what is displayed on screen.

File: cubic-server/interface/LogsInterface.cpp [hunk 1]
```diff
@@ -4,23 +4,64 @@
...
-  m_VBox(Gtk::ORIENTATION_VERTICAL)
+  m_VBox_main(Gtk::ORIENTATION_VERTICAL),
+  m_VBox_logs(Gtk::ORIENTATION_VERTICAL),
+  m_HBox_filters(Gtk::ORIENTATION_HORIZONTAL),
+  m_Info("Info"),
+  m_Warn("Warning"),
+  m_Error("Error"),
+  m_Fatal("Fatal"),
+  m_Debug("Debug"),
+  m_Reset("↺")
```
Comment (Trompettesib) [inline cubic-server/interface/LogsInterface.cpp:18]: no sure about this character for reseting filters but I'm not a designer so I will not say anything ^^

File: cubic-server/interface/LogsInterface.hpp [hunk 2]
```diff
@@ -3,19 +3,32 @@
...
+#include "logging/Logger.hpp"
+
+class LogsInterface : public Gtk::Box {
     public:
         LogsInterface();
         virtual ~LogsInterface();
-        bool on_timeout();
 
     private:
+        bool on_log_to_display();
```
Comment (Trompettesib) [inline cubic-server/interface/LogsInterface.hpp:14]: function names aren't in pascalCase

**Review by Trompettesib — APPROVED** [review]
Some things are cringe like the nessessity of using a new logLevel but overall it's good

**Review by STMiki — APPROVED** [review]

**Review by huntears — CHANGES_REQUESTED** [review]
View scrolls up and down continuously

**Review by STMiki — DISMISSED** [review]

**Review by Trompettesib — APPROVED** [review]

**Review by STMiki — APPROVED** [review]

**Review by huntears — APPROVED** [review]

**Pull request merged**